src/web/volta.py

```diff
@@ -11,6 +12,8 @@
     'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.51 Safari/537.36',
 }
 
+#Creo la carpeta donde guardare los archivos .html creados
+Path('paginas').mkdir(exist_ok=True)
```

Comment:
Esta bien usar `pathlib`, es la nueva forma de hacer las cosas que hacemos con `os`
Pero este archivo esta usando `os` y es mejor por ahora conservar esto.
La sugerencia aquí es hacerlo al modo de `os`:

```
path = 'paginas'
if not os.path.exists(path):
    os.makedirs(path)
```

Pero no aquí, este código suelto (no dentro de ninguna función) se va a ejecutar más veces de las que queremos.
Lo mejor es colocar este código donde realmente lo necesitamos: en la función `scrape_volta` antes del for que itera por todas las páginas.